pkg/util/throttle.go

```diff
@@ -26,9 +26,12 @@ type RateLimiter interface {
 	Accept()
 	// Stop stops the rate limiter, subsequent calls to CanAccept will return false
 	Stop()
+	// Saturation returns a percentage number which describes how saturated
+	// this rate limiter is.
```

Comment:
@gmarek I don't really follow how that would make a difference. There's no difference between "burst" and "regular" tokens.